system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -26,19 +26,47 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
+;; Helper constants and utility functions
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(def var1-name "variable1")
+(def var1-name-new "variable1-new")
+(def del-var-name "deleted-variable")
+(def user-name "user1")
+
+(defn- get-tokens
```

Comment:
There is no permission checks on retrieving variable concepts. Can we remove the token from this test? The `variable/grant-all-variable-fixture` call should have given guest user all the necessary permissions.